setup.cfg

```
@@ -22,6 +22,7 @@ disallow_untyped_defs = True
 ignore_missing_imports = True
 files =
     kombu/utils/time.py
```

I guess the comma is needed 🤔 
```suggestion
    kombu/utils/time.py,
```